InnerEye/ML/Histopathology/models/deepmil.py

```diff
@@ -201,6 +201,10 @@ def _shared_step(self, batch: Dict, batch_idx: int, stage: str) -> Dict[ResultsK
                         ResultsKey.PROB: probs, ResultsKey.PRED_LABEL: preds,
                         ResultsKey.TRUE_LABEL: bag_labels, ResultsKey.BAG_ATTN: bag_attn_list,
                         ResultsKey.IMAGE: batch[TilesDataset.IMAGE_COLUMN]})
+        if (TilesDataset.TILE_X_COLUMN in batch.keys()) and (TilesDataset.TILE_Y_COLUMN in batch.keys()):
+            results.update({ResultsKey.TILE_X: batch[TilesDataset.TILE_X_COLUMN],
+                           ResultsKey.TILE_Y: batch[TilesDataset.TILE_Y_COLUMN]}
+                           )
         return results
```

Comment:
Maybe it is good to put a clear assert in the else here?

Reply:
not sure I interpret correctly: would you like this to be wrapped in a try/exception assertion or just adding a else where we raise maybe a warning that coordinates are not found?

Reply:
the latter

Reply:
done!